pandora-calculation/src/main/resources/test1.properties

```diff
@@ -0,0 +1,28 @@
+dic.enabled=true
```

Comment:
specific test1.properties should not be required for the issue in the ticket

Reply:
Deleted test1.properties